### PR TITLE
Feature/nmd 306 implement google authorization

### PR DESCRIPTION
**Pull request by hinkevich**

### CODE REVIEW

File: IdentityServer/IdentityServer/Config.cs [hunk 1]
```diff
@@ -95,21 +82,6 @@ public static class Config
                         IdentityServerConstants.StandardScopes.OpenId,
                         IdentityServerConstants.StandardScopes.Profile
                     }
-                },
-
-                new Client
```
Comment (moltenessence) [inline IdentityServer/IdentityServer/Config.cs:100]: why you delete swagger client?

File: IdentityServer/IdentityServer/Config.cs [hunk 2]
```diff
@@ -50,19 +50,6 @@ public static class Config
                     AllowedScopes = { "News.API" }
                 },
                 new Client
-                {
-                    ClientId = "client_id",
```
Comment (moltenessence) [inline IdentityServer/IdentityServer/Config.cs:54]: Why do you delete client with client credentials grant type?

Comment (hinkevich) [reply IdentityServer/IdentityServer/Config.cs:54]: this is duplicated code, after conflict resolution

File: IdentityServer/IdentityServer/Config.cs (hunk 1 above)
Comment (hinkevich) [reply IdentityServer/IdentityServer/Config.cs:100]: this is duplicated code, after conflict resolution

**Review by stas1408123 — APPROVED** [review]

**Review by Genresal — APPROVED** [review]

**Review by blackberts — APPROVED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

**Pull request merged**